src/pages/App/App.test.tsx

```diff
@@ -33,7 +33,7 @@ test("routes correctly and renders key pages", async () => {
 });
 
 test("renders error page when route is not found", () => {
-  const badRoute = "/not-a-route";
+  const badRoute = "/help/not-a-route";
```

Comment:
What was the issue with the previous bad route?

Reply:
Design choices! I wanted "/investing" rather than something like, "/categories/investing", so the code is `/:category`. But `/:category` matches `/not-a-route`, which is why I needed to adjust the `badRoute`.

I considered calling it `/weight-:category` because I expect that in the future, rather than give the user only content that matches the category, it will be a weight-based logic. But since, as Spring, I want _the specific information I'm looking for_, that choice felt inconsistent.

Reply:
Weight-based == the app changing the mix of content based on the user's activities and interests we have, so it may not necessarily be all one category.